Examples/BookingSystem.AspNetCore/Stores/OrderStore.cs

```diff
@@ -35,7 +37,7 @@ public override async Task<bool> CustomerCancelOrderItems(OrderIdComponents orde
         {
             try
             {
-                return await FakeBookingSystem.Database.CancelOrderItems(
+                return await _fakeBookingSystem.database.CancelOrderItems(
```

Comment:
.NET convention is properties start with upper case

```suggestion
                return await _fakeBookingSystem.Database.CancelOrderItems(
```